ansible/roles/stack-sunbird/templates/dial-service_application.conf

```diff
@@ -188,5 +188,6 @@ jsonld {
     type = "sb"
     localPath = "/tmp"
     ttl = 300
+    sb_schema = ["https://sunbirdstagingpublic.blob.core.windows.net/sunbird-dial-staging/jsonld-schema/local/sb/schema.jsonld"]
```

Comment:
Please removed hardcoded blob url and variablise the storage account and container name

Reply:
@G33tha  This will be a public URL. That is yet to be done. We have discussed it with SanthoshG and Rayulu. Once that is done, conf will be updated. Till then, we need this for testing the module.